README.md

```diff
@@ -309,6 +307,11 @@ $ curl -X POST -H "Authorization: token=$AUTH_TOKEN" http://<dcos_url>/service/c
 4. Once the restore operation is finished, check if the data is restored correctly.
 5. [Uninstall](https://github.com/mesosphere/dcos-cassandra-service#uninstall) old cluster.
 
+**Upgrade Notes:** 
+* The upgrade process will cause all of your Cassandra node processes to restart.
+* Upgrades are supported from version `N` to `N + 1`. So, if you are on version `N` and want to get to version `N + 2`, please upgrade from version `N` to `N + 1` first, followed by an upgrade from `N + 1` to `N + 2`.
+* Upgrade from `1.0.15-3.0.7` to `1.0.16-3.0.8` is not supported.
```

Comment:
Can they still perform the backup/restore upgrade? Saying 'not supported' implies that's also not possible.